network/encoding.go

```diff
@@ -78,6 +78,16 @@ func RegisterMessage(msg Message) MessageTypeID {
 	return msgType
 }
 
+// RegisterMessages is a convenience function to register multiple messages
+// together.
```

Comment:
We could add in the comment 1) what is the returned value, and 2) specifying what happens in case multiple `Message` are the same. Namely nothing, the same ids will be returned multiple times.